packages/redwood/src/index.ts

```diff
@@ -237,9 +284,30 @@ export const build: BuildV2 = async ({
     lambdaOutputs[outputName] = lambda;
   }
 
+  // Older versions of redwood did not create 200.html automatically
+  // From v0.50.0+ 200.html is always generated as part of web build
+  // Note that in builder post-processing, we remove the .html extension
+  const fallbackHtmlPage = existsSync(join(webDistPath, '200'))
```

Comment:
Since we wouldn't be renaming the source file on the filesystem:

```suggestion
  const fallbackHtmlPage = existsSync(join(webDistPath, '200.html'))
```